astropy/io/ascii/cds.py

```diff
@@ -126,8 +126,8 @@ def get_cols(self, lines):
                 col.type = self.get_col_type(col)
 
```

Comment:
One suggestion I have here is to add a comment describing the regex in brief or at least mentioning its capability (like nullability, order specifier, description). This is from a development perspective; if I ever have to make changes to this code/regex and have to understand each and every part of the regex I might get lost as to why elements were added but the comment might help. This isn't essential, but might be helpful.